Functions/xASL_qc_CollectQC_ASL.m

```diff
@@ -94,10 +95,17 @@
 		Path_pWM = x.P.Path_rc2T1;
 	end
 	
+    % Read CBF NIfTI
     pGM = xASL_io_Nifti2Im(Path_pGM);
     pWM = xASL_io_Nifti2Im(Path_pWM);
-    imCBF = xASL_io_Nifti2Im(x.P.Path_CBF);
-
+    if isfield(x.Q,'bUseBasilQuantification') && x.Q.bUseBasilQuantification==1
```

Comment:
@MDijsselhof The way it was, the output has two different paths, `CBF` and `CBF_Basil`. We may want to think this through before changing everything. If with `bUseBasilQuantification`, we want to do the whole pipeline based on the BASIL images, we might as well rename the Basil output to `qCBF` rather than `qCBF_BASIL`, to avoid unnecessary complexity?

Also, best to define as much as these paths etc at the beginning of the ASL module